### PR TITLE
Load correctly when tree item is not in active namespace

### LINKED ISSUE #267: Explorer: Load pods not in non-active namespaces
As @itowlson had noted, selecting a Pod in a non-default namespace from the Node tree in the explorer won't load the resource in VSCode. This is due to the way the `Load` command works (by using the current active namespace). This should be fixed such that the `Load` command is aware of the namespace resources are in, so it can load them.

### PR DESCRIPTION
**Pull request by itowlson**
Fixes #267.

### CODE REVIEW

File: src/explorer.ts [hunk 1]
```diff
@@ -188,6 +189,10 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         this.resourceId = kind.abbreviation + '/' + id;
     }
 
+    namespace(): string | null {
```
Comment (bnookala) [inline src/explorer.ts]: can we turn this into a getter? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

File: src/explorer.ts [hunk 2]
```diff
@@ -188,6 +189,10 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
...
+        return (this.metadata && this.metadata.namespace) ? this.metadata.namespace : null;
```
Comment (bnookala) [inline src/explorer.ts]: is it even possible for an object to not have a namespace?

File: src/kuberesources.virtualfs.ts [hunk 3]
```diff
@@ -51,7 +51,9 @@ export class KubernetesResourceVirtualFileSystemProvider implements FileSystemPr
     async loadResource(uri: Uri): Promise<string> {
         const outputFormat = workspace.getConfiguration('vs-kubernetes')['vs-kubernetes.outputFormat'];
         const value = querystring.parse(uri.query).value;
-        const sr = await this.kubectl.invokeAsyncWithProgress(`-o ${outputFormat} get ${value}`, `Loading ${value}...`);
+        const ns = querystring.parse(uri.query).ns;
```
Comment (bnookala) [inline src/kuberesources.virtualfs.ts]: querystring.parse returns an object, so you should just be able to do this once and read from the object.

File: src/explorer.ts (hunk 2 above)
Comment (itowlson) [reply src/explorer.ts]: It _is_ possible per the spec, though shouldn't happen on a get... but in any case `metadata` is optional and untyped so I don't want to presume anything.  I want to get it clearer and more consistent what the tree view nodes contain, and that might obviate this check, but for the time being I feel safer with it in!

**Pull request merged**